ark/phenotyping/som_utils.py

```diff
@@ -855,7 +941,7 @@ def train_cell_som(fovs, base_dir, pixel_consensus_dir, cell_table_name,
 
     # generate a matrix with each fov/cell label pair with their pixel SOM/meta cluster counts
     print("Counting the number of pixel SOM/meta cluster counts for each fov/cell pair")
-    cluster_counts = compute_cell_cluster_counts(
+    cluster_counts = create_c2pc_data(
         fovs, consensus_path, cell_table_path, cluster_col
     )
 
```

Comment:
I think it makes sense to do the cell size normalization here, before writing to file, instead of doing it twice in both "create_cell_som.R" and "run_cell_som.R"

Reply:
@cliu72 so the tricky thing here is we need the un-normalized pixel SOM/meta counts data per cell for computing weighted channel average but the normalized pixel SOM/meta counts data per cell for training the SOM and assigning the SOM labels back. That's why I have it this way.

For now, I'll can save both as intermediate files.